src/shared/components/banner-react-conf/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import styles from './style.css';
+import { imageAssetsEndpoint } from '../../config';
+
+export default class BannerReactConf extends Component {
+
+  render () {
+    return (
+      <div className={styles.bannerContainer}>
+        <a href="http://react.london" target="_blank">
+          <img
+            src={imageAssetsEndpoint + 'react-conf-desktop-banner.png'}
+            className={styles.reactConfDesktopBanner}
+          />
+          <img
+            src={imageAssetsEndpoint + 'react-conf-tablet-banner.png'}
+            className={styles.reactConfTabletBanner}
+          />
+          <img
+            src={imageAssetsEndpoint + 'react-conf-mobile-banner.png'}
+            className={styles.reactConfMobileBanner}
+          />
```

Comment:
Would it be better to use [`srcset`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Example_4_Using_the_srcset_and_sizes_attributes) here instead of CSS?
Looks like it's [quite well supported](http://caniuse.com/#feat=srcset) now